walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google LLC
+// Copyright 2020 Google LLC
 //
```

Comment:
Ok in this case, but generally it's a best practices to either:

- Have a PR to just update the copyrights across the whole codebase, or
- Update the copyright only in the files you were touching otherwise in this PR

Reply:
Got it! I'll revert the files I didn't touch.